desc/backend.py

```diff
@@ -6,7 +6,7 @@
 from desc import set_device, config as desc_config
 
 # we divide by zero in a few places but then overwrite with the
-# correct values, so lets suppress annoying warnings about that
+# correct values, so let's suppress annoying warnings about that
```

Comment:
maybe a comment for @f0uriest  and @ddudt  but where do we do this in the code?

Reply:
it might be a holdover from when we had compute functions at the axis. We would compute things using the regular methods (giving nan at the axis due to 1/g) then replace the axis values with the correct ones using lhopital. We could probably remove this

Reply:
That is correct, this was from the axis values. Should we keep this suppressed? I would like to add the axis values back in eventually, but that's low priority. And suppressing these warnings could be bad if we are missing other instances where they would appear.

Reply:
Just removed this, we can add it back in if needed, or better yet do it locally where we know we're doing it